content/questions/template-literals/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Template Literals
+tags:
+  - compiling
+  - scripting
+  - ei11 
+  - babel
+order: 48
+date: Sat Oct 12 2019 17:08:35 GMT-0400 (Eastern Daylight Time)
+answers: 
+  - ecmascript 
+  - es2015 // correct
+  - stage-6
+  - es6
```

Comment:
Isn't es6 the same thing as es2015?

Reply:
@nas5w Interesting ~ I'd say it is referred to as the same thing but supporting IE versions is different. No IE support is available for es6 however Partial support is available for template literals in es2015. 

I'd believe the correct answer is still es2015 as it was the first implementation of the template literal syntax into JS.  

-----
es6: https://caniuse.com/#feat=template-literals - No support for template literals in IE 6-11
es2015: https://caniuse.com/#feat=es6 - Partial support for template literals in IE 6-11
reference: http://www.ecma-international.org/ecma-262/6.0/#sec-template-literals
history: https://www.benmvp.com/learning-es6-history-of-ecmascript/